techniques/fileDistribution/copyGitFile/1.3/copyFileFromSharedFolder.st

```diff
@@ -110,13 +110,13 @@ bundle agent download_from_shared_folder
         ifvarclass => "!execute_command_${index}";
 
       "@@copyFile@@result_success@@${copyfile[${index}][uuid]}@@Post-modification hook@@${copyfile[${index}][name]}@@${g.execRun}##${g.uuid}@#${copyfile[${index}][destination]} was already in the desired state, so no command was executed"
-        ifvarclass => "execute_command_${index}.copy_file_${index}_kept.!copy_file_${index}_modified";
+        ifvarclass => "execute_command_${index}.!copy_file_${index}_failed.!copy_file_${index}_modified.copy_file_${index}_kept";
 
       "@@copyFile@@result_success@@${copyfile[${index}][uuid]}@@Post-modification hook@@${copyfile[${index}][name]}@@${g.execRun}##${g.uuid}@#The post-hook command for ${copyfile[${index}][destination]} was correctly executed"
-        ifvarclass => "copyfile_posthook_${index}_command_run_ok";
+        ifvarclass => "execute_command_${index}.copyfile_posthook_${index}_command_run_ok";
 
       "@@copyFile@@result_error@@${copyfile[${index}][uuid]}@@Post-modification hook@@${copyfile[${index}][name]}@@${g.execRun}##${g.uuid}@#The post-hook command for ${copyfile[${index}][destination]} couldn't be executed"
-        ifvarclass => "copyfile_posthook_${index}_command_run_failed";
+        ifvarclass => "execute_command_${index}.copyfile_posthook_${index}_command_run_failed";
```

Comment:
These changes are not strictly necessary but complete the combinatory if/not logic of the classes in this set of reports, making them a LOT easier to understand.